src/shared/components/blocks/subscribe/Subscribe.module.css

```diff
@@ -1,3 +1,149 @@
+@import "../../../styles/colors";
+@import "../../../styles/custom-medias";
+@import "../../../styles/mixins/typography";
+
 .subscribe {
-    background: rosybrown;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--color-celtic);
+}
+
+.joinUs {
+    position: relative;
+    flex: 1;
+    text-align: center;
+}
+
+.firstText {
+    @mixin typography-h1 rem;
+    color: var(--color-tan);
+}
+
+.secondText {
+    width: 408px;
+    @mixin typography-body-1 rem;
+    margin: 70px auto 0 auto;
+    color: var(--color-white);
+    font-size: 18px;
+}
+
+.image {
+    height: 100%;
+    flex: 1;
+    color: var(--color-tan);
+
+    & div svg {
+        width: 75% !important;
+        height: 75% !important;
+    }
+}
+
+form {
+    position: absolute;
+    left: 50%;
+    margin-top: 40px;
+    transform: translateX(-50%);
+    display: flex;
+    flex-direction: row;
+}
+
+.input {
+    @mixin typography-body-1 rem;
+    width: 270px;
+    height: 60px;
+    padding-left: 20px;
+    border: 2px solid var(--color-tan);
+    background: var(--color-celtic);
+    color: var(--color-white);
+}
+
+.submit {
+    width: 180px;
+    height: 60px;
+}
+
+.inputSuccess {
+    border: 2px solid #2d473a;
+}
+
+.success {
+    padding-right: 20px !important;
+    display: flex !important;
+    flex-direction: column !important;
+    align-items: flex-end !important;
+    background: none !important;
```

Comment:
do you really need to use `!important`?
try to have a bigger target like `.subscribe form .success` for example